tests/framework/db/mssql/DBLibPDOTest.php

```
@@ -0,0 +1,83 @@
+ * @since 22.0
+ */
+#[Group('db')]
+#[Group('mssql')]
+#[Group('pdo')]
+final class DBLibPDOTest extends DatabaseTestCase
+{
+    protected $driverName = 'sqlsrv';
+
+    public function testGetAttributeReturnsServerVersion(): void
+    {
+        $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)